tests/rptest/tests/data_transforms_test.py

```diff
@@ -672,4 +684,214 @@ def test_tunable_configs(self):
+            assert metrics_samples is not None
+            return sorted(metrics_samples.keys()) == sorted(
+                self.LOGGER_METRICS)
+
+        assert any(
+            [
+                has_logger_metrics(n, MetricsEndpoint.METRICS)
+                for n in self.redpanda.nodes
+            ]
+        ), "Expected some node to export logger metrics on the internal endpoint"
+
+        assert any(
+            [
+                has_logger_metrics(n, MetricsEndpoint.PUBLIC_METRICS)
+                for n in self.redpanda.nodes
+            ]
+        ), "Expected some node to export logger metrics on the internal endpoint"
+
+    @cluster(num_nodes=3)
+    def test_logger_metrics_values(self):
+        n_xform = len(self.topics) // 2
+
+        its = []
+        for i in range(0, n_xform):
+            its.append(
+                self.setup_identity_xform(
+                    self.topics[2 * i],
+                    self.topics[2 * i + 1],
+                    name=f"logger{i+1}",
+                )[0])
+        N_PER_TP = 100
+        for i in range(0, N_PER_TP):
+            for it in its:
+                self._rpk.produce(
+                    it.name,
+                    self.random_ascii_string(10, 100),
+                    self.random_ascii_string(10, 100),
+                )
+
+        def get_total_events_per_xform() -> tuple[dict[str, int], int]:
+            totals = {}
+            dropped = 0
+            for node in self.redpanda.nodes:
+                samples = self.unpack_samples(
+                    self.get_metrics_from_node(
+                        node,
+                        self.LOGGER_METRICS,
+                        endpoint=MetricsEndpoint.METRICS))
+                for m in samples['events_total']:
+                    xfm_name = m['labels']['transform_name']
+                    totals[xfm_name] = totals.get(xfm_name, 0) + m['value']
+                dropped += sum(
+                    [m['value'] for m in samples["events_dropped_total"]])
+            self.logger.debug(json.dumps(totals, indent=1))
+            return (totals, dropped)
+
+        def get_total_events() -> tuple[int, int]:
+            tot = 0
+            totals, dropped = get_total_events_per_xform()
+            for k in totals:
+                tot += totals[k]
+            self.logger.debug(f"Cluster total log events: {tot}")
+            return (tot, dropped)
+
+        n_events_expected = N_PER_TP * len(its)
+        wait_until(lambda: get_total_events()[0] == n_events_expected,
+                   timeout_sec=30,
+                   backoff_sec=5,
+                   err_msg=f"never got all the events")
+
+        totals, dropped = get_total_events_per_xform()
+        assert len(totals) == len(its)
+        for name in totals:
+            assert totals[
+                name] == N_PER_TP, f"Expected {N_PER_TP} log events from transform {name}, got{totals[name]}"
```

Comment:
This is a brittle test, transforms at at least once and can be executed multiple times (that is valid) so we need greater than or equal here

Reply:
fixed